packages/perseus/src/widgets/expression/expression.tsx

```diff
@@ -414,81 +417,6 @@ const styles = StyleSheet.create({
     },
 });
 
-/**
- * Determine the keypad configuration parameters for the input, based on the
- * provided properties.
- *
- * There are two configuration parameters to be passed to the keypad:
- *   (1) The keypad type. For the Expression widget, we always use the
- *       Expression keypad.
- *   (2) The extra keys; namely, any variables or constants (like Pi) that need
- *       to be included as keys on the keypad. These are scraped from the answer
- *       forms.
- */
-export const keypadConfigurationForProps = (
```

Comment:
Most of this logic was for deriving extra keys, so I just transformed this into `deriveExtraKeys`.